src/nodes/QskArcRenderer.cpp

```diff
@@ -48,4 +52,28 @@ void QskArcRenderer::renderArc(const QRectF& rect,
 
     painter->setPen( QPen( brush, metrics.width(), Qt::SolidLine, Qt::FlatCap ) );
     painter->drawArc( rect, metrics.startAngle(), metrics.spanAngle() );
+
+    if( borderMetrics.width( Qsk::Inner ) > 0 )
+    {
+        // draw inner border:
+        const qreal i = metrics.width() / 2;
+        const auto innerRect = rect.marginsRemoved( { i, i, i, i } );
+        const QColor innerColor = borderColors.color( Qsk::Inner );
+
+        painter->setPen( QPen( innerColor, borderMetrics.width( Qsk::Inner ),
+            Qt::SolidLine, Qt::FlatCap ) );
+        painter->drawArc( innerRect, metrics.startAngle(), metrics.spanAngle() );
```

Comment:
We just draw additional arcs, because QPainter uses the pen to draw arcs, and there doesn't seem to be a way to draw borders with a pen. Otherwise we would have to call drawPie(), but that would introduce other difficulties.